app/controllers/categories_controller.rb

```diff
@@ -6,6 +6,5 @@ def index
 
   def show
     @category = Category.find_by(id: params[:id])
-    @projects = @category.projects
   end
```

Comment:
I think we can delete this show method and all reference to category show paths.